Applications/SlicerApp/Testing/Python/SubjectHierarchyCorePluginsSelfTest.py

```diff
@@ -126,7 +126,7 @@ def section_MarkupRole(self):
     studyNode = vtkMRMLSubjectHierarchyNode.CreateSubjectHierarchyNode(slicer.mrmlScene, patientNode, self.studyName, slicer.vtkMRMLSubjectHierarchyConstants.GetDICOMLevelStudy())
     markupsShNode = vtkMRMLSubjectHierarchyNode.CreateSubjectHierarchyNode(slicer.mrmlScene, studyNode, slicer.vtkMRMLSubjectHierarchyConstants.GetDICOMLevelSeries(), self.sampleMarkupName, markupsNode)
 
-    self.assertTrue( markupsShNode != None )
+    self.assertTrue( markupsShNode is not None )
```

Comment:
This could go further by writing ```self.assertIsNotNone(markupsShNode)```. But there are a lot of other cases, so this might beyond scope of this PR. For example, in the lines below, where we could write ```assertEqual(a, b)``` in place of ```assertTrue(a == b)```. Using the more specific functions usually gives more useful error messages.